cli/data.py

```diff
@@ -83,26 +90,16 @@ def update(aggregate_to_country: bool, state: Optional[str], fips: Optional[str]
     """Updates latest and timeseries datasets to the current checked out covid data public commit"""
     path_prefix = dataset_utils.DATA_DIRECTORY.relative_to(dataset_utils.REPO_ROOT)
 
-    data_source_classes = set(
-        chain(
-            chain.from_iterable(ALL_FIELDS_FEATURE_DEFINITION.values()),
-            chain.from_iterable(ALL_TIMESERIES_FEATURE_DEFINITION.values()),
-        )
-    )
-    data_sources = {
-        data_source_cls.SOURCE_NAME: data_source_cls.local().multi_region_dataset()
-        for data_source_cls in data_source_classes
-    }
+    timeseries_field_datasets = build_field_datasets(ALL_TIMESERIES_FEATURE_DEFINITION)
```

Comment:
fwiw, I have a hard time parsing `build_field_datasets` - mostly I didn't know what `field_datasets` was (without thinking about it).  How about something like `load_datasets_by_field`, `load_dataset_list_by_field` or `load_multiregion_datasets_by_field`?

Reply:
Done